src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { useAppSelector } from "../redux/hooks";
+import { getProfile } from "../services/sdk";
+import { Profile } from "../utilities/types";
+import { ListStatus } from "../utilities/enums";
+
+const FollowersFollowing = (): JSX.Element => {
+  const graph = useAppSelector((state) => state.user.graph);
+  const [selectedListTitle, setSelectedListTitle] = useState<ListStatus>(
+    ListStatus.CLOSED
+  );
+
+  const [usersList, setUsersList] = useState<Profile[] | any>([]);
```

Comment:
I believe the state for this should always be only `Profile[]`. Adding `any` subverts adding a typing.

I'd consider renaming this to `selectedList` to correspond with `selectedListTitle`. It helps clarify it's purpose. When I first read this variable I thought it was a list for the user which was confusing.

Reply:
i like this. thanks!